lua/nvim-treesitter/parsers.lua

```diff
@@ -43,11 +43,21 @@ local list = setmetatable({}, {
     filetype_to_parsername[parserconfig.filetype or parsername] = parsername
   end,
 })
+list.agda = {
+  install_info = {
+    url = "https://github.com/AusCyberman/tree-sitter-agda",
+    branch = "master",
+    files = { "src/parser.c", "src/scanner.cc" },
+  },
+  filetype = "agda",
+  maintainers = { "@AusCyberman" },
```

Comment:
please ask them first whether they are OK with being listed as maintainer for _nvim-treesitter_!

Reply:
I have asked him..

Reply:
And did they agree?

Reply:
I will wait for his reply

Reply:
To be clear, this is just about listing them as the maintainer. The parser can be added even without a maintainer listed.

Reply:
Typically the person adding the parser cares about this language and becomes contact person for nvim-treesitter

Reply:
@Decodetalkers friendly ping. Can you fix this?

Reply:
Sorry, fix what?

Reply:
> @Decodetalkers friendly ping. Can you fix this?

Sorry, fix what?

Reply:
to put you as maintainer instead of the person who wrote the parser

Reply:
I see

Reply:
> to put you as maintainer instead of the person who wrote the parser

Now changed..